lib/fs-tree/entries.js

```diff
@@ -0,0 +1,98 @@
+'use strict';
+
+function Entries(entries) {
```

Comment:
this should likely just be expressed as an explicit object, rather then implicitly with closures.